.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+- If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
+- Please label this pull request according to what type of issue you are addressing, especially if this is a release targeted pull request. For reference on required PR/issue labels, read here:
```

Comment:
labels are not supported.. if they are it's only for maintainers..

Reply:
Indeed they are supported here (see https://github.com/kubernetes-sigs/cri-tools/pull/636#issuecomment-654039996), I think a valid kind should be applied by the author in any case.

Reply:
Thx! i'm used to clicking on the github label button :-) will try out `/kind label`

Reply:
/resolved

Reply:
i wonder if there is a `/` action to show resolved :-)